File: ordenamiento po seleccion/seleccion.cs
using System;

class Programa {
    static void OrdenamientoPorSeleccion(int[] lista) {
        for (int i = 0; i < lista.Length - 1; i++) {
            int indiceMinimo = i;
            for (int j = i + 1; j < lista.Length; j++) {
                if (lista[j] < lista[indiceMinimo]) {
                    indiceMinimo = j;
                }
            }
            int temporal = lista[i];
            lista[i] = lista[indiceMinimo];
            lista[indiceMinimo] = temporal;
        }
    }

    static void MostrarLista(int[] lista) {
        Console.WriteLine(string.Join(", ", lista));
    }

    static void Main() {
        int[] numeros = {64, 25, 12, 22, 11};

        Console.WriteLine("Lista original:");
        MostrarLista(numeros);

        OrdenamientoPorSeleccion(numeros);

        Console.WriteLine("Lista ordenada:");
        MostrarLista(numeros);
    }
}

File: ordenamiento po seleccion/seleccion.js
function ordenamientoPorSeleccion(lista) {
    for (let i = 0; i < lista.length; i++) {
        let indiceMinimo = i;
        for (let j = i + 1; j < lista.length; j++) {
            if (lista[j] < lista[indiceMinimo]) {
                indiceMinimo = j;
            }
        }
        [lista[i], lista[indiceMinimo]] = [lista[indiceMinimo], lista[i]];
    }
}

function mostrarLista(lista) {
    console.log(lista.join(", "));
}


let numeros = [64, 25, 12, 22, 11];

console.log("Lista original:");
mostrarLista(numeros);

ordenamientoPorSeleccion(numeros);

console.log("Lista ordenada:");
mostrarLista(numeros);